src/spec/Difference/Difference.spec.jsx

```diff
@@ -0,0 +1,103 @@
+import { render, screen, cleanup, fireEvent } from "@testing-library/react";
```

Comment:
P1:  render, screen, fireEvent, cleanup 순으로 정렬해주세요~!

```suggestion
import { render, screen,  fireEvent, cleanup } from "@testing-library/react";
```